src/index.html

```diff
@@ -3,10 +3,460 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="catalog">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li
+            class="nav__item"
+          >
+            <a class="is-active nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="hover"
+          >
+            <a class="nav__link" data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <article class="card first" data-qa="card">
```

Comment:
```suggestion
      <article class="card" data-qa="card">
```
![image](https://user-images.githubusercontent.com/96409563/205493932-e201c96d-28b0-42cd-ac70-4038ea0996ca.png)
First is not a separate BEM block. You can use pseudoclasses or a BEM modifier if you need to provide some styles for the first article.